ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.10.yaml

```diff
@@ -0,0 +1,19 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```

Comment:
Let's make this 1.14.

Reply:
juchuuuuuu - good, thanks

I will bump (dif PR) for master too, than